# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsIFrame.cpp
# Commit: a79d9a152a6d
# Full Hash: a79d9a152a6d52a71210d4675d3c86b94615cd1b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-08-04 21:45:54
# Regressor Bug: 1722300
# File Overlap Count: 1
# Description:
#   Bug 1722300 - patch 4 - Implement internal destinations when generating PDF output through cairo. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120960
# ==============================================================================

diff -r b11f7f6bff04 -r a79d9a152a6d layout/generic/nsIFrame.cpp
--- a/layout/generic/nsIFrame.cpp	Wed Aug 04 12:52:37 2021 +0000
+++ b/layout/generic/nsIFrame.cpp	Wed Aug 04 12:52:38 2021 +0000
@@ -4004,8 +4004,8 @@
   Maybe<nsDisplayListBuilder::Linkifier> linkifier;
   if (StaticPrefs::print_save_as_pdf_links_enabled() &&
       aBuilder->IsForPrinting()) {
-    linkifier.emplace(aBuilder, aChild);
-    linkifier->MaybeAppendLink(aBuilder, aChild, aLists.Content());
+    linkifier.emplace(aBuilder, aChild, aLists.Content());
+    linkifier->MaybeAppendLink(aBuilder, aChild);
   }
 
   nsIFrame* child = aChild;